### PR TITLE
HW-51229: List PayPal accounts

### PR DESCRIPTION
**Pull request by vshcherbyna-epam**

### CODE REVIEW

File: core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListPayPalAccountsTest.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+    @Rule
+    public HyperwalletMockWebServer mServer = new HyperwalletMockWebServer();
+    @Rule
+    public HyperwalletSdkMock mSdkMock = new HyperwalletSdkMock(mServer);
+    @Rule
+    public MockitoRule mMockito = MockitoJUnit.rule();
+    @Mock
+    private HyperwalletListener<HyperwalletPageList<PayPalAccount>> mListener;
+    @Captor
+    private ArgumentCaptor<HyperwalletPageList<PayPalAccount>> mListTransferMethodCaptor;
```
Comment (fmattos-hw) [inline core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListPayPalAccountsTest.java]: mListPayPalCaptor

File: core/src/test/resources/errors/pay_pal_accounts_error_response.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (fmattos-hw) [inline core/src/test/resources/errors/pay_pal_accounts_error_response.json]: remove the underscore from the name, it should be paypal_...

File: core/src/test/resources/pay_pal_accounts_response.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (fmattos-hw) [inline core/src/test/resources/pay_pal_accounts_response.json]: same here paypal

File: core/src/test/resources/pay_pal_no_accounts_response.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (fmattos-hw) [inline core/src/test/resources/pay_pal_no_accounts_response.json]: here as well paypal...

File: core/src/main/java/com/hyperwallet/android/model/HyperwalletPayPalAccountPagination.java [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+import java.util.Date;
+import java.util.Map;
+
+/**
+ * Represents the PayPal Account pagination fields
+ */
+public class HyperwalletPayPalAccountPagination extends HyperwalletTransferMethodPagination {
+
+    protected static final String TRANSFER_METHOD_CREATE_ON = "createdOn";
```
[the remark this answers is not in the document]
Comment (vshcherbyna-epam) [reply core/src/main/java/com/hyperwallet/android/model/HyperwalletPayPalAccountPagination.java]: added

File: core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListPayPalAccountsTest.java [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+        RecordedRequest recordedRequest = mServer.getRequest();
+        verify(mListener).onSuccess(mListTransferMethodCaptor.capture());
+        verify(mListener, never()).onFailure(any(HyperwalletException.class));
+
+        HyperwalletPageList<PayPalAccount> payPalAccountsResponse = mListTransferMethodCaptor.getValue();
+
+        assertThat(payPalAccountsResponse.getCount(), is(2));
+        assertThat(payPalAccountsResponse.getDataList(), hasSize(2));
+        assertThat(payPalAccountsResponse.getOffset(), is(0));
+        assertThat(payPalAccountsResponse.getLimit(), is(10));
```
[the remark this answers is not in the document]
Comment (vshcherbyna-epam) [reply core/src/test/java/com/hyperwallet/android/transfermethod/HyperwalletListPayPalAccountsTest.java]: added

**Pull request merged**